daffodil-tdml-lib/src/main/scala/org/apache/daffodil/tdml/TDMLRunner.scala

```diff
@@ -1639,59 +1634,53 @@ case class UnparserTestCase(ptc: NodeSeq, parentArg: DFDLTestSuite)
       case e: Exception => throw TDMLException(e, implString)
     }
 
-    val diagnostics = {
```

Comment:
This is mostly an indentation change, I recommend hiding whitespace when viewing this diff.